### DISCUSSION TITLE
gcc-11 / remove libgcc_s.1.1.dylib to make binaries more portable

### DISCUSSION THREAD

**Discussion by 21tesla** [Homebrew/discussions#2623] [Everyday usage]
I have compiled some software with the **gcc-11**, **g++-11** and **gfortran-11** compilers. 

I have noticed that there is dependency on _/opt/homebrew/opt/gcc/lib/gcc/11/libgcc_s.1.1.dylib_ 

Can this dependency be removed somehow to make binaries more portable?

**Comment by Bo98** [comment]
Try `-static-libgcc`.

**Reply by JasonShiao** [reply]
Hi, I installed `opencv` with brew install and it depends on the `openblas-0.3.20` which further depends on gcc 11
How can I force the brew-installed `openblas` to link with `-static-libgcc` as you mentioned here?

**Reply by carlocab** [reply]
You can’t do that with software you didn’t build yourself, I’m afraid.

If you really need a version of `openblas` that does this you can create your own version of the `openblas` formula that does this. But note that `brew` won’t use your custom `openblas` to fulfil `opencv`‘s dependency on `openblas` —- you’ll also need your own custom `opencv` formula if this is what you are looking for.
